### ISSUE TITLE
Edition files do not contain geometries of objects that were moved

### ISSUE THREAD

**Issue by planemad** [planemad/osm-history-processor#10]
Picked this up today as there was a mismatch in the size of the edition files of users and the objects visible in JOSM.

After some investigation @geohacker and I found that if a user moves an existing node and does not create any new ids, the geometries of these changes are not available in the edition files for peer review.

We need to change the [osmfilter](http://wiki.openstreetmap.org/wiki/Osmfilter#Keep_specific_Objects_and_their_Dependents) flags so that it retains these geometries.

cc @rub21

**Comment by maning** [comment]
For sidewalk tagging, we only touch tags not geometry.  The current files do not include node geoms, so we cannot view the actual data in JOSM.

**Comment by planemad** [comment]
@maning thats right, the current process cannot fetch geometries for retagging projects like sidewalks since the replication files do not contain them. Fetching each geometry referenced using the API also does not sound like a scalable solution.

The immediate fix I see is to use Overpass to extract the data required if its not too large.
